pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1563,8 +1563,8 @@ assertNoAdditions {
 
   kulala-nvim = super.kulala-nvim.overrideAttrs {
```

Comment:
This one is actually depending on a migration https://github.com/mistweaverco/kulala.nvim/issues/790 https://github.com/mistweaverco/kulala.nvim/blob/9fc4831a116fb32b0fd420ed483f5102a873446a/lua/kulala/config/init.lua#L49